R/logistics_helper.R

```diff
@@ -0,0 +1,72 @@
+## log-likelihood
+logit_loglik <- function(beta, X, Y){
+  p <- 1 / (1 + exp(-X %*% beta))
+  l <- -(t(Y) %*% log(p) + t(1 - Y) %*% log(1 - p))
+  return(l)
+}
+
+
+## gradient
```

Comment:
-1

These comments don't describe any more than the function names, if you don't want to write full function documentation I'd just remove them.

Reply:
there are two other isntances of this in the file that are missed